app/javascript/packs/shared/data_grid/DataGrid.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import ReactDataGrid from 'react-data-grid';
+
+class DataGrid extends Component {
```

Comment:
I was thinking about inheritance, but in React.js it's way [more powerful](https://facebook.github.io/react/docs/composition-vs-inheritance.html) to use composition instead of inheritance.